src/libertem/io/fs.py

```diff
@@ -19,7 +19,12 @@ def _access_ok(path):
 
 
 def _get_alt_path(path):
-    cur_path = Path(path).resolve()
+    try:
+        cur_path = Path(path).resolve()
+    # Triggered by empty DVD drive on Windows
+    except PermissionError:
```

Comment:
Hm, on Linux, `resolve()` doesn't trigger `PermissionError` - is it possible that this will break alternative path suggestions on Windows if just a sub directory is not accessible? For example, `C:/my-data/no-access/` should not suggest the home directory, but `C:/my-data/`

Reply:
Ok, I'll test what happens with a folder where the user doesn't have rights vs. a device that is not ready so that the scenario you describe is handled gracefully